pigeon.py

```diff
@@ -17,6 +21,9 @@ def handler(event, context):
             host=host,
             port=port,
             virtual_host=virtual_host,
+            connection_attempts=10,
+            socket_timeout=10,
+            retry_delay=1,
```

Comment:
This means that the lambda function will retry a few times. This will alleviate rabbitmq ephemeral problems.

I totally pulled these values out of thin air. I'm game for better ones.